src/main/java/namor/template/calculator/CalculatorConsoleApplication.java

```diff
@@ -0,0 +1,42 @@
+package namor.template.calculator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class CalculatorConsoleApplication {
+
+    public static void main(String args[]) {
+
+        double result;
+        BufferedReader bufferedReader;
+        CalculatorFactory calculatorFactory = new CalculatorFactory();
+        Calculator calculator;
+
+        bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+        try {
+
+            while(true) {
+
+                System.out.println("Please, enter something!");
+                String input = bufferedReader.readLine();
+
+                if ("q".equals(input)) {
+                    System.out.println("End program!");
+                    System.exit(0);
+                } else if (input.isEmpty()) {
+                    System.out.println("0");
```

Comment:
main method is not correct place to return result. Only calculator responsible for returning result even if it's empty string which is 0

Reply:
so better to extract all from main method to private class or to move it to the other class?

Reply:
else if (input.isEmpty()) {
           System.out.println("0");
}

This part of a code should not be in main method